hooks/post_gen_project.py

```diff
@@ -1,24 +1,52 @@
+from pathlib import Path
+import os
 
-def gitlab_ci_enabled(adapter):
-    if adapter == "snowflake":
-        print(
-            "Set the following environment variables in your GitLab project to run dbt as part of your CI pipeline:\n"
-            " - DBT_ACCOUNT: Your snowflake account\n"
-            " - DBT_USER: Your snowflake username\n"
-            " - DBT_PASSWORD: Your snowflake password\n"
-            " - TEST_DATABASE: The Database name you use for running CI jobs\n\n"
-            "Learn more about GitLab CI/CD variables here: https://docs.gitlab.com/ee/ci/variables/"
-        )
+PROJECT_DIRECTORY = Path().resolve()
+def move_user_profiles():
+  source_file = PROJECT_DIRECTORY.joinpath('.config/user_profiles.yml')
+  target_file = PROJECT_DIRECTORY.joinpath('profiles.yml')
+  Path(source_file).rename(target_file)
 
+def initialize_git_repo():
+  os.system("git init")
+
+def post_init_messages(adapter, ci_tool):
+
+  if ci_tool == "GitHub":
+    docs = "https://docs.github.com/en/actions/reference/environment-variables"
+  elif ci_tool == "GitLab":
+    docs = "https://docs.gitlab.com/ee/ci/variables/"
+
+  print(
+    "\nProject initialized successfully!\n"
+  )
+
+  if ci_tool == "None":
+    message = (
+      f"A base profile.yml file has been created in folder your project folder\n"
+      f"Copy it to ~/.dbt and edit its contents\n"
+      f"For more information see https://docs.getdbt.com/dbt-cli/configure-your-profile \n\n"
+    )
+  else:
+    message = (
+      f"A base profile.yml file has been created in folder your project folder\n"
+      f"Copy it to ~/.dbt and edit its contents for local development\n"
+      f"For more information see https://docs.getdbt.com/dbt-cli/configure-your-profile \n\n"
+
+      f"The CI job will leverage the profile.yml file in the .config folder.\n"
+      f"You will need to set the corresponding environment variables in the {ci_tool}\n"
+      f"project to run dbt as part of your CI pipeline.\n"
+      f"Learn more about {ci_tool} CI/CD variables here: {docs}"
+    )
+  print(message)
 
 def main():
-    print("\nProject initialized successfully!\n")
-    
     adapter = "{{ cookiecutter.adapter }}"
+    ci_tool = "{{ cookiecutter.ci_tool }}"
 
-    if "{{ cookiecutter.ci_tool }}" == "GitLab":
-        gitlab_ci_enabled(adapter)
-
+    move_user_profiles()
+    initialize_git_repo()
```

Comment:
I'd ask for confirmation on this. This project might be already a part of an existing git repo.